backend/go.mod

```diff
@@ -156,3 +157,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace go.vallahaye.net/connect-gateway => github.com/birdayz/connect-gateway v0.0.0-20230929163733-c35a55537a3e
```

Comment:
Do we want to continue with this fork or did you make an upstream PR? If so, what had to be changed there, that we are using this fork?

Reply:
yes i did open an upstream PR here: https://github.com/vallahaye/connect-gateway/pull/26

it loses errordetails currently, the PR fixes this.